src/plugins/vis_type_timeseries/server/validation_telemetry/validation_telemetry_service.ts

```diff
@@ -1,97 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import { CoreSetup, Plugin, PluginInitializerContext } from 'kibana/server';
-import { UsageCollectionSetup } from '../../../usage_collection/server';
-import { tsvbTelemetrySavedObjectType } from '../saved_objects';
-
-export interface ValidationTelemetryServiceSetup {
-  logFailedValidation: () => void;
-}
-export interface Usage {
-  failed_validations: number;
-}
-
-export class ValidationTelemetryService implements Plugin<ValidationTelemetryServiceSetup> {
-  private kibanaIndex: string = '';
-  async setup(
-    core: CoreSetup,
-    {
-      usageCollection,
```

Comment:
as far as I see, the `usageCollection` is not used anywhere in tsvb anymore -> can be removed from `optionalPlugins`